addons/medical/stringtable.xml

```diff
@@ -1031,6 +1031,15 @@
             <Chinesesimp>大量失血中</Chinesesimp>
             <Chinese>大量失血中</Chinese>
         </Key>
+        <Key ID="STR_ACE_Medical_Status_Lost_Blood2">
+            <English>Lost some blood Class II Hemorrhage</English>
```

Comment:
Haven't seen how it looks in game, but it seems odd to me that the class directly runs on from the description in these strings